agent/jvm/src/test/java/org/jolokia/jvmagent/JolokiaServerTest.java

```diff
@@ -294,7 +293,20 @@ public void sslWithSpecialHttpsSettings() throws Exception {
 
                 try {
                     TrustManager tms[] = getTrustManagers(true);
-                    SSLContext sc = SSLContext.getInstance(protocol);
+                    SSLContext sc;
+                    try {
+                        sc = SSLContext.getInstance(protocol);
+                    } catch (NoSuchAlgorithmException e) {
+                        // on the IBM JVM v8 we get an exception on SSLv3 which fails the entire test, safe-guarding for it.
```

Comment:
Any idea why this fails for IBM ? If this is inevitable I would check also for the IBM VM and only continue if its on IBM (and fail like before for Oracle).

Reply:
Yeah, now that I look at it again I believe this should be changed (partially because the reason given was what I thought in the beginning to be the reason but later forgot to change it before sending the pull request). From what I found out SSLv3 is simply disabled, which is a bit confusing because the list of cipher suites all start with SSL_, but what's TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA on OpenJDK is SSL_ECDHE_RSA_WITH_AES_128_CBC_SHA on IBM.